src/EnergyPlus/HVACFan.cc

```diff
@@ -1169,6 +1169,16 @@ namespace HVACFan {
         }
     }
 
+    int FanSystem::fanNumOfSpeed() const
+    {
+        return m_numSpeeds;
+    }
+
+    Real64 FanSystem::AirMassFlowRateAtSpeed(int const SpeedNum)
+    {
+        return m_massFlowAtSpeed[SpeedNum];
+    }
```

Comment:
If m_numSpeeds and m_massFlowAtSpeed were public you wouldn't need these new functions.